src/components/SocialBtns/SocialBtns.js

```diff
@@ -1,24 +1,32 @@
+/* global FB, location */
 import React, { PropTypes } from 'react';
 import FaFacebookSquare from 'react-icons/lib/fa/facebook-square';
 import FaTwitter from 'react-icons/lib/fa/twitter';
 import styles from './SocialBtns.sass';
 
 const SocialBtns = ({
-  fbLink = 'https://www.facebook.com',
-  twitterLink = 'https://twitter.com',
+  fbLink = `https://www.facebook.com/sharer/sharer.php?u=${location.href}`,
```

Comment:
<img width="859" alt="screen shot 2017-04-10 at 12 08 16 pm" src="https://cloud.githubusercontent.com/assets/4714249/24871184/cfebfaca-1de6-11e7-803d-b7a1fdd6aeca.png">

Can you include meta tags in the head to include title, image and description?

```
 <meta property="og:url"           content="http://www.your-domain.com/your-page.html" />
  <meta property="og:type"          content="website" />
  <meta property="og:title"         content="Your Website Title" />
  <meta property="og:description"   content="Your description" />
  <meta property="og:image"         content="http://www.your-domain.com/path/image.jpg" />
```